plugin/trino-hive/src/main/java/io/trino/plugin/hive/coercions/TimestampCoercer.java

```diff
@@ -44,6 +47,7 @@
             .appendLiteral(' ')
             .append(ISO_LOCAL_TIME)
             .toFormatter()
+            .withResolverStyle(ResolverStyle.STRICT)
```

Comment:
if i remove this line, which test will show me  that we're no longer compatible with Hive behavior?

Reply:
Nice catch !! Have added a `2019-02-29 23:59:59.123` to the invalid set of values where it would fail.